js/app.js

```diff
@@ -0,0 +1,71 @@
+// Selectors for counters
+const TOTAL = 'total-zombies';
+const DEAD = 'dead-zombies';
+const SMALL = 'small-zombies';
+const MAD = 'mad-zombies';
+const STRONG = 'strong-zombies';
+
+let counters = {
+    [TOTAL]: 0,
+    [DEAD]: 0,
+    [SMALL]: 0,
+    [MAD]: 0,
+    [STRONG]: 0,
+};
+
+let calculateZombies = () => {
+    let totalCounter = 0;
+
+    zombiesData.map((zombieHP) => {
+        let zombieType = getZombieType(zombieHP);
+        let counter = counters[zombieType];
+
+        counters[zombieType] = ++counter;
```

Comment:
please do not use pre-increment
what's the point of using 

```
let counter = counters[zombieType];
counters[zombieType] = ++counter;

```
?

you just can use simple increment like "anyCounter++"

Reply:
fixed